packages/ocap-kernel/src/remotes/network.ts

```diff
@@ -223,6 +223,10 @@ export async function initNetwork(
   }
 
   const SCTP_USER_INITIATED_ABORT = 12; // see RFC 4960
+  // Connection retry/backoff configuration
+  const MAX_CONNECT_ATTEMPTS = 5;
+  const BASE_BACKOFF_MS = 1_000; // 1s
+  const MAX_BACKOFF_MS = 30_000; // 30s
```

Comment:
These appear to be the same values as default in `retry.ts`.  Do we want to hard code them here also?  And should these values possibly be stored in the database with some interface for setting them?